dotcom-rendering/src/web/components/ArticleContainer.tsx

```diff
@@ -109,6 +109,42 @@ const articleAdStyles = css`
 	.ad-slot--fluid {
 		width: 100%;
 	}
+	.ad-slot--liveblog-inline {
+		margin: 0 auto ${space[3]}px;
+
+		.ad-slot__label {
+			color: ${neutral[46]};
+			border-top-color: ${neutral[86]};
+		}
+
+		&.ad-slot--outstream {
+			${from.tablet} {
+				width: 620px;
```

Comment:
I'm not certain what this width is doing but liveblogs have a width of `700px`, not `620px` - does this need to be accounted for here as well?

See: https://github.com/guardian/dotcom-rendering/blob/oliver/instagram-island/dotcom-rendering/src/web/layouts/LiveLayout.tsx#L175

Reply:
This width is to account for [outstream ads](https://github.com/guardian/commercial-core/blob/main/src/ad-sizes.ts#L70) which have a width of `620px`.

Reply:
> This width is to account for [outstream ads](https://github.com/guardian/commercial-core/blob/main/src/ad-sizes.ts#L70) which have a width of `620px`.

Ah, I see. Thanks for checking!